src/components/Tile/Tile.js

```diff
@@ -1,9 +1,40 @@
 const React = require("react");
+const blockedLinks = require("lib/blockedLinks");
+const {connect} = require("react-redux");
+const actions = require("actions/index");
 
 const Tile = React.createClass({
+  /**
+   * Blocks the site (removes it from the grid) and calls the given callback
+   * when done.
+   */
+  block: function (e) {
+    e.preventDefault();
+    if (!this.isBlocked()) {
+      blockedLinks.block(this.props.url);
+      if (this.props.dispatch) {
+        // Dispatch doesn't exist in a test environment.
```

Comment:
I agree. @k88hudson I imagine this is a common issue given what we talked about last week with having a separate export of the `Tile` object that isn't 'reduxy' to be used for testing with refs. Any idea if there is a better way to use the non-redux export without having to do what I'm doing here?